hail/src/main/scala/is/hail/expr/ir/functions/ReferenceGenomeFunctions.scala

```diff
@@ -85,5 +85,11 @@ class ReferenceGenomeFunctions(rg: ReferenceGenome) extends RegistryFunctions {
           Seq(TString(), TInt32())).get
         If(isValid(Array(contig, pos)), getRef(Array(contig, pos, before, after)), NA(TString()))
     }
+
+    registerRGCode("contigLength", TString(), TInt32(), null) {
```

Comment:
what's the deal with this `pt` argument which is `null` and not used in the definition of `registerRGCode`?

Reply:
this is the ptype registration function. If `null`, the function returns the canoncical PType for the virtual type.